azurelinuxagent/ga/update.py

```diff
@@ -320,10 +320,14 @@ def run(self, debug=False):
 
             #
             # Initialize the goal state; some components depend on information provided by the goal state and this
-            # call ensures the required info is initialized (e.g telemetry depends on the container ID.)
+            # call ensures the required info is initialized (e.g. telemetry depends on the container ID.)
             #
             protocol = self.protocol_util.get_protocol()
-            protocol.client.update_goal_state(force_update=True)
```

Comment:
Removed the `force_update=True` as its not really needed (this being the first GS fetch).